### ISSUE TITLE
[Application Settings] User should be able to access a settings menu to configure application

### ISSUE THREAD

**Issue by vykster** [OpenHousing/hhs#8]
Settings include:

- Date range in application - #9 
- Break points for user types for CJ and HMIS - #11 #12 
- Fields or Columns displayed in application.

**Comment by KClough** [comment]
To confirm, I believe we discussed moving these to environment variable configurations for now. The end user will not be able to update, but anyone with access to where the application is running can reconfigure/restart the app to change these settings.

**Comment by vykster** [comment]
@KClough yes. Moving these to config is best option right now, @tplagge and I will edit as necessary on deploy or restart app(s).
